### DISCUSSION TITLE
No autotagging

### DISCUSSION THREAD

**Discussion by scialo** [garethgeorge/backrest#287] [General]
how set no auto tagging but set only own tag?
backuprest 1.0.0

**Comment by garethgeorge** [comment]
Hey, sorry this isn't a supported feature. The tags Backrest adds are fundamental to it's operation. You can add extra tags by adding a `--tag your-tag` flag if you like.

Backrest uses the tags it adds to reconstruct your snapshots into plans if you restore onto a new computer with the same config.

**Comment by scialo** [comment]
ok, thanks
